upgrade/kind.go

```diff
@@ -99,6 +99,11 @@ func GetRepoKind(ctx Context, repo ProviderRepo) (*GoMod, error) {
 	} else if !ok {
 		return nil, fmt.Errorf(bridgeMissingMsg)
 	}
+	pf, ok, err := originalGoVersionOf(ctx, repo, filepath.Join("provider", "go.mod"), "github.com/pulumi/pulumi-terraform-bridge/pf")
+	if (err != nil || !ok) && ctx.UpgradePfVersion {
```

Comment:
`!ok` is different then an `err != nil`. If we get an error, we should report it and abort. `!ok` just means it wasn't found. This is expected behavior for lots of our providers.